innopoints/views/account.py

```diff
@@ -69,12 +68,12 @@
 def subquery_to_events(subquery, event_type):
     """Take a subquery that has an 'entry_time' field and output a query
     that packs the rest of the fields into a JSON payload and returns it with the time."""
-    payload = db.func.row_to_json(as_row(subquery)).cast(JSONB) - 'entry_time'
+    # payload = db.func.row_to_json(query).cast(JSONB) - 'entry_time'
     return db.session.query(
-        'entry_time',
+        subquery.c.entry_time.label('entry_time'),
         db.literal(event_type).label('type'),
-        payload.label('payload')
-    ).select_from(subquery)
+        (db.func.row_to_json(db.literal_column(subquery.name)).cast(JSONB) - 'entry_time').label('payload')
```

Comment:
How did inlining `payload` fix anything?

Reply:
It's not only a matter of inlining